contracts/stake/src/state.rs

```diff
@@ -34,23 +34,33 @@ use crate::*;
 /// valid stake.
 #[derive(Debug, Default, Clone)]
 pub struct StakeState {
-    stakes: BTreeMap<[u8; BlsPublicKey::SIZE], (StakeData, StakeKeys)>,
     burnt_amount: u64,
+    conf: StakeConfig,
     previous_block_state:
         BTreeMap<[u8; BlsPublicKey::SIZE], (Option<StakeData>, BlsPublicKey)>,
+    stakes: BTreeMap<[u8; BlsPublicKey::SIZE], (StakeData, StakeKeys)>,
 }
 
 const STAKE_CONTRACT_VERSION: u64 = 8;
 
 impl StakeState {
     pub const fn new() -> Self {
         Self {
-            stakes: BTreeMap::new(),
             burnt_amount: 0u64,
+            conf: StakeConfig::new(),
             previous_block_state: BTreeMap::new(),
+            stakes: BTreeMap::new(),
         }
     }
 
+    pub fn config(&self) -> &StakeConfig {
+        &self.conf
```

Comment:
I think these should have the same name. Either the method needs to be called `conf` or the field-name is `config`. I have a slight preference for the latter.

Reply:
I also prefer naming it config

Reply:
2vs1

I'll change it